src/main/resources/verily_secret.json

```diff
@@ -1,11 +1,11 @@
 {
   "installed": {
-    "client_id": "590500654757-l8ripukp65bpdr7kc7nq0o4bv8kmtku0.apps.googleusercontent.com",
     "project_id": "terra-ots0-prod",
     "auth_uri": "https://accounts.google.com/o/oauth2/auth",
     "token_uri": "https://oauth2.googleapis.com/token",
     "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
-    "client_secret": "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",
```

Comment:
Does this need to be removed from Git history too or are you going to rotate the creds?

Reply:
Ideally this needs to be rotated, however rotating ti will break existing installations. There is a long term plan however, sharing it offline